samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +346,49 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	cJSON *msg_obj = NULL;
+
+	/* Send off a hello world message! */
+	/* This does not match a predefined schema. But that's not a problem! */
+	msg_obj = cJSON_CreateObject();
+	if (msg_obj == NULL) {
+		LOG_ERR("Failed to create cJSON object");
+		return -EINVAL;
+	}
+
+	if (!cJSON_AddStringToObject(msg_obj, "sample_message",
+					"Hello World, from the REST Device Message Sample!")) {
+		LOG_ERR("Failed to add sample message to cJSON object");
+		return -EINVAL;
+	}
+
+	if (IS_ENABLED(CONFIG_REST_DEVICE_MESSAGE_TEST_BUILD))	{
+		/* Get the current timestamp*/
+		int64_t time_now;
+
+		err = date_time_now(&time_now);
+		if (err) {
+			LOG_ERR("Failed to get timestamp for Hello World message");
+			return err;
+		}
+
+		if (!cJSON_AddNumberToObject(msg_obj, NRF_CLOUD_MSG_TIMESTAMP_KEY, (double)time_now)) {
```

Comment:
You could continue the if statement on the next line.  I suggest doing so after `NRF_CLOUD_MSG_TIMESTAMP_KEY,`.